ionoscloud/test_constants.go

```diff
@@ -1580,6 +1580,74 @@ resource ` + constant.ServerVCPUResource + ` ` + constant.ServerTestResource + `
 }`
 )
 
+const testAccCheckServerVCPUShutDown = `
+resource ` + constant.DatacenterResource + ` ` + constant.DatacenterTestResource + ` {
+  name       = "server-test"
+  location   = "de/txl"
+}
+resource "ionoscloud_ipblock" "webserver_ipblock" {
+  location = ` + constant.DatacenterResource + `.` + constant.DatacenterTestResource + `.location
+  size = 4
+  name = "webserver_ipblock"
+}
+resource ` + constant.LanResource + ` ` + constant.LanTestResource + ` {
+  datacenter_id = ` + constant.DatacenterResource + `.` + constant.DatacenterTestResource + `.id
+  public = true
+  name = "public"
+}
+resource ` + constant.ServerVCPUResource + ` ` + constant.ServerTestResource + ` {
+  name = "` + constant.ServerTestResource + `"
+  datacenter_id = ` + constant.DatacenterResource + `.` + constant.DatacenterTestResource + `.id
+  vm_state = "SHUTOFF"
```

Comment:
```suggestion
  vm_state = ` + cloudapiserver.EnterpriseServerStop + `
```

Reply:
but please test that it works

Reply:
plan definitions in tests will reference the constants instead of hardcoding